media-video/h264enc/h264enc-10.4.7-r1.ebuild

```diff
@@ -30,6 +31,8 @@ RDEPEND="
 	vorbis? ( media-sound/vorbis-tools )"
 DEPEND=""
 
+PATCHES=( "${FILESDIR}"/libaacplusenc.patch )
```

Comment:
Please rename the patch including the package version where this was introduced. Ie h264enc-10.4.1-libaacplusenc.patch